### PR TITLE
LPS-183241 Adding Accounts and User-accouts to Site-Initializer

### PR DESCRIPTION
**Pull request by EugenioAntunes**
[LPS-183142](https://issues.liferay.com/browse/LPS-183142)

### CODE REVIEW

**Review by Elvinho — APPROVED** [review]

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/src/main/resources/site-initializer/accounts.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+[
```
Comment (welitonpe) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/src/main/resources/site-initializer/accounts.json:1]: Replace names for no real persons

**Review by rafaelfs98 — APPROVED** [review]
LGTM :fire:

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/src/main/resources/site-initializer/accounts.json (hunk 1 above)
Comment (EugenioAntunes) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/src/main/resources/site-initializer/accounts.json:1]: Done!

**Review by welitonpe — APPROVED** [review]
Good Job!

**Review by kevenleone — APPROVED** [review]
LGTM 🔥

**Comment by kevenleone** [comment]
ci:forward

**Comment by kevenleone** [comment]
ci:stop

**Comment by EugenioAntunes** [comment]
ci:test:sf

**Comment by EugenioAntunes** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
ci:forward

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/src/main/resources/site-initializer/accounts.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+	{
+		"externalReferenceCode": "TESTRAY-001",
+		"name": "Lizeth Turcotte",
```
[the remark this answers is not in the document]
Comment (EugenioAntunes) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/src/main/resources/site-initializer/accounts.json]: Done!

**Comment by EugenioAntunes** [comment]
ci:forward

**Comment by EugenioAntunes** [comment]
ci:forward

**Comment by EugenioAntunes** [comment]
ci:forward

**Pull request closed without merging**